apps/manager-v2/src/app/manage-accounts/account-details/configuration/configuration.component.ts

```diff
@@ -37,6 +38,7 @@ export class ConfigurationComponent implements OnInit, OnDestroy {
   isSaving = false;
 
   defaultLimits?: AccountTypeDefaults;
+  created = '';
```

Comment:
Is this property needed ?

Reply:
No, I did a test and forgot to remove it. good catch